docs/content/direct/combined-status.md

```diff
@@ -94,6 +96,22 @@ When there are N "GROUP BY" columns, the result has a row for each tuple of valu
 
 See `types.go`.
 
+### Queryable Objects
+
+A CEL expression within a `StatusCollector` can reference the following objects:
+
+1. `inventory`: The inventory object for the workload object:
+   - `inventory.name`: The name of the inventory object.
+
+1. `obj`: The workload object from the WDS:
+   - All fields of the workload object except the status subresource.
+
+1. `returned`: The reported state from the WEC:
+   - `returned.status`: The status section of the object returned from the WEC.
+
+1. `propagation`: Metadata about the end-to-end propagation process:
+   - `propagation.lastReturnedUpdateTimestamp`: Time of last update to any returned state.
```

Comment:
Someone writing an expression using this will need to know exactly what sort of "time" this is.